database/migrations/2020_01_13_000000_create_orders_table.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateOrdersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('orders', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('reference')->unique()->index();
+            //$table->unsignedInteger('courier_id')->index();
+            $table->unsignedBigInteger('customer_id')->index()->nullable();
+
+            if (Schema::hasTable(config('user_table'))) {
```

Comment:
This is likely going to fail due to the lack of a configuration file.

Reply:
Oops, nice catch. That should be `config('user_table', 'users')`; I'll fix it now.

Reply:
also, the syntax is wrong for the config path